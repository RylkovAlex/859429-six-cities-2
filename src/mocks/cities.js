export const cityMock = {
  name: `Amsterdam`,
  location: {
    latitude: 52.370216,
    longitude: 4.895168,
    zoom: 10,
  },
};

export const citiesMock = [
  {
    name: `1`,
    location: {
      latitude: 1,
      longitude: 2,
      zoom: 3,
    },
  },
  {
    name: `2`,
    location: {
      latitude: 1,
      longitude: 2,
      zoom: 3,
    },
  },
];
