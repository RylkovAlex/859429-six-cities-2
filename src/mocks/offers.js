export const offerCards = ([
  {
    city: {
      name: `Amsterdam`,
      location: {
        latitude: 52.370216,
        longitude: 4.895168,
        zoom: 10
      }
    },
    id: 1,
    previewImage: `img/apartment-01.jpg`,
    images: [`../img/room.jpg`, `../img/apartment-01.jpg`, `../img/apartment-02.jpg`, `../img/apartment-03.jpg`, `../img/studio-01.jpg`, `../img/apartment-01.jpg`],
    title: `Beautiful & luxurious studio at great location`,
    isFavorite: false,
    isPremium: false,
    rating: 4.8,
    type: `apartment`,
    bedrooms: 3,
    maxAdults: 4,
    price: 120,
    goods: [`Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
    host: {
      id: 3,
      isPro: true,
      name: `Angelina`,
      avatarUrl: `../img/avatar-angelina.jpg`,
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    location: {
      latitude: 52.3909553943508,
      longitude: 4.85309666406198,
      zoom: 8
    }
  },
  {
    city: {
      name: `Amsterdam`,
      location: {
        latitude: 52.370216,
        longitude: 4.895168,
        zoom: 10
      }
    },
    id: 2,
    previewImage: `img/apartment-02.jpg`,
    images: [`../img/room.jpg`, `../img/apartment-01.jpg`, `../img/apartment-02.jpg`, `../img/apartment-03.jpg`, `../img/studio-01.jpg`, `../img/apartment-01.jpg`],
    title: `Beautiful & luxurious studio at great location`,
    isFavorite: false,
    isPremium: true,
    rating: 4.8,
    type: `apartment`,
    bedrooms: 3,
    maxAdults: 4,
    price: 120,
    goods: [`Wi-Fi`, `Heating`, `Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
    host: {
      id: 3,
      isPro: true,
      name: `Angelina2`,
      avatarUrl: `../img/avatar-angelina.jpg`,
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    location: {
      latitude: 52.369553943508,
      longitude: 4.85309666406198,
      zoom: 8
    }
  },
  {
    city: {
      name: `Amsterdam`,
      location: {
        latitude: 52.370216,
        longitude: 4.895168,
        zoom: 10
      }
    },
    id: 3,
    previewImage: `img/apartment-03.jpg`,
    images: [`../img/room.jpg`, `../img/apartment-01.jpg`, `../img/apartment-02.jpg`, `../img/apartment-03.jpg`, `../img/studio-01.jpg`, `../img/apartment-01.jpg`],
    title: `Beautiful & luxurious studio at great location`,
    isFavorite: true,
    isPremium: true,
    rating: 4.8,
    type: `apartment`,
    bedrooms: 3,
    maxAdults: 4,
    price: 120,
    goods: [`Kitchen`, `Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
    host: {
      id: 3,
      isPro: true,
      name: `Angelina3`,
      avatarUrl: `../img/avatar-angelina.jpg`,
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    location: {
      latitude: 52.3909553943508,
      longitude: 4.929309666406198,
      zoom: 8
    }
  },
  {
    city: {
      name: `Amsterdam`,
      location: {
        latitude: 52.370216,
        longitude: 4.895168,
        zoom: 10
      }
    },
    id: 4,
    previewImage: `img/apartment-01.jpg`,
    images: [`../img/room.jpg`, `../img/apartment-01.jpg`, `../img/apartment-02.jpg`, `../img/apartment-03.jpg`, `../img/studio-01.jpg`, `../img/apartment-01.jpg`],
    title: `Beautiful & luxurious studio at great location`,
    isFavorite: false,
    isPremium: true,
    rating: 4.8,
    type: `apartment`,
    bedrooms: 3,
    maxAdults: 4,
    price: 120,
    goods: [`Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
    host: {
      id: 3,
      isPro: true,
      name: `Angelina4`,
      avatarUrl: `../img/avatar-angelina.jpg`,
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    location: {
      latitude: 52.3809553943508,
      longitude: 4.939309666406198,
      zoom: 8
    },
  },
  {
    city: {
      name: `Cologne`,
      location: {
        latitude: 50.938903,
        longitude: 6.966074,
        zoom: 10
      }
    },
    id: 5,
    previewImage: `img/apartment-01.jpg`,
    images: [`../img/room.jpg`, `../img/apartment-01.jpg`, `../img/apartment-02.jpg`, `../img/apartment-03.jpg`, `../img/studio-01.jpg`, `../img/apartment-01.jpg`],
    title: `Beautiful & luxurious studio at great location`,
    isFavorite: false,
    isPremium: true,
    rating: 4.8,
    type: `apartment`,
    bedrooms: 3,
    maxAdults: 4,
    price: 120,
    goods: [`Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
    host: {
      id: 3,
      isPro: true,
      name: `Angelina4`,
      avatarUrl: `../img/avatar-angelina.jpg`,
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    location: {
      latitude: 50.938903,
      longitude: 6.966074,
      zoom: 8
    },
  },
  {
    city: {
      name: `Paris`,
      location: {
        latitude: 48.851353,
        longitude: 2.341412,
        zoom: 10
      }
    },
    id: 6,
    previewImage: `img/apartment-01.jpg`,
    images: [`../img/room.jpg`, `../img/apartment-01.jpg`, `../img/apartment-02.jpg`, `../img/apartment-03.jpg`, `../img/studio-01.jpg`, `../img/apartment-01.jpg`],
    title: `Beautiful & luxurious studio at great location`,
    isFavorite: false,
    isPremium: true,
    rating: 4.8,
    type: `apartment`,
    bedrooms: 3,
    maxAdults: 4,
    price: 120,
    goods: [`Cable TV`, `Washing machine`, `Coffee machine`, `Dishwasher`],
    host: {
      id: 3,
      isPro: true,
      name: `Angelina4`,
      avatarUrl: `../img/avatar-angelina.jpg`,
    },
    description: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    location: {
      latitude: 48.851353,
      longitude: 2.341412,
      zoom: 8
    },
  },

]);
