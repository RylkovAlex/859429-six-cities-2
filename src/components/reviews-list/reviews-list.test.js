import React from 'react';
import renderer from 'react-test-renderer';
import ReviewsList from './reviews-list';
import {reviewsMock} from '../../mocks/reviews';
jest.mock(`../review-item/review-item.jsx`, () => () => <div/>);

it(`ReviewsList correctly renders after relaunch`, () => {
  const props = {
    reviews: reviewsMock,
  };

  const tree = renderer
    .create(<ReviewsList {...props}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
