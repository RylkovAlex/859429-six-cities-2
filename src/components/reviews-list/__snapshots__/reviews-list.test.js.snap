// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsList correctly renders after relaunch 1`] = `
<ul
  className="reviews__list"
/>
`;
